app/views/batch_contexts/_new_bc_form.erb

```diff
@@ -1,4 +1,14 @@
-<h3>Complete the form below</h3>
+
+<div data-controller="globus">
+
+<div class="row">
+    <div class="col">
+        <h3>Complete the form below</h3>
+    </div>
+    <div class="col" style="text-align: right;">
+      <button class="btn btn-sul-dlss" data-globus-target="requestGlobusLink" data-action="click->globus#createDestination">Request Globus Link</button>
+    </div>
+</div>
```

Comment:
The new Request Globus Link button. Note I needed to enclose the whole form in a div in order to set the StimulusController on it, which allows it to update the Staging Location after the Globus Link was created.